### ISSUE TITLE
implode() variable order deprecated error when creating checkout object

### ISSUE THREAD

**Issue by brianherbert** [coinbase/coinbase-commerce-php#24]
PHP 7.4.4

Attempting to create a checkout object and I'm having a deprecation error thrown.

```php
    use CoinbaseCommerce\ApiClient;
    use CoinbaseCommerce\Resources\Checkout;

    $checkout_data = [
        'name' => '1 of a thing',
        'description' => 'Tokens for use on Qoin Live.',
        'pricing_type' => 'fixed_price',
        'local_price' => [
            'amount' => 10,
            'currency' => 'USD'
        ],
        'requested_info' => []
    ];
    $checkout_obj = Checkout::create($checkout_data);
```

This produces the following error in `vendor/coinbase/coinbase-commerce/src/Util.php:63` (joinPath()):

```
implode(): Passing glue string after array is deprecated. Swap the parameters
```

**Comment by brianherbert** [comment]
Scratch this, it looks like it was already fixed with #18. Looks like there hasn't been a release cut in a while so installing via composer doesn't include this fix.

**Comment by omidahourai** [comment]
@brianherbert we released it shortly after you posted this, sorry for the delay!
